api/controller/logs.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package percona.platform.dbaas.controller.v1beta1;
+
+option go_package = "controller;controllerv1beta1";
+
+// Log contains log for certain pod's container.
+message Log {
```

Comment:
Not sure, but that feels a bit strange to have a singular form there

Reply:
Yeah, it might be better especially if we use `repeated string` for log field. The field would then transform to plural form, too -> `repeated string logs`

Reply:
Done